main.py

```diff
@@ -2,6 +2,43 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    create_genres(("Western", "Action", "Dramma"))
+    create_actors([("George", "Klooney"), ("Kianu", "Reaves"),
+                   ("Scarlett", "Keegan"), ("Will", "Smith"),
+                   ("Jaden", "Smith"), ("Scarlett", "Johansson")])
+    update()
+    delete()
+    return get_actor()
+
+
+def create_genres(genres_name: tuple) -> None:
+    for name in genres_name:
+        Genre.objects.create(name=name)
+
+
+def create_actors(actors: list[tuple[str, str]]) -> None:
+    for actor in actors:
+        first_name, last_name = actor
```

Comment:
You can skip this line by doing unpacking in a for loop and save yourself one line of code:
`for first_name, last_name in actors'